tests/pages/Landing.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import Landing from '../../src/pages/Landing';
+import { describe, it, expect } from 'vitest';
+
+describe('Landing Page', () => {
+  it('renders main sections and content', () => {
+    render(<Landing />);
+
+    expect(screen.getByTestId('hero-section')).toBeInTheDocument();
+    expect(screen.getByTestId('hero-title')).toHaveTextContent(
+      'Welcome to Feature-Flag',
+    );
+    expect(screen.getByAltText('RDS Logo')).toBeInTheDocument();
+
+    expect(screen.getByTestId('what-is-section')).toBeInTheDocument();
+    expect(screen.getByTestId('why-use-section')).toBeInTheDocument();
+
+    expect(screen.getByTestId('features-section')).toBeInTheDocument();
+    expect(screen.getByTestId('features-grid')).toBeInTheDocument();
```

Comment:
no test cases for scenarios where constants like `FEATURES` are empty or missing. this might cause rendering issues in the `features-grid`.

Reply:
This would have a been a good case to handle if we were fetching the `FEATURES` array from server. But is a UI content we are saving in constant and using here.

`FEATURES` is coming from your constants file (`../constants/landing`) and contains hardcoded content, it will never be empty in practice.

Adding test for such scenarios doesn't do any good but adds unnecessary complexity in test, burden to maintain and Doesn't provide any real value or confidence in our code.

It renders this part: we don't have content to show even if this doesn't exist it should be empty
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/eaa6dff8-1744-4150-a131-701c8b8a3123">

Reply:
In case we want to be sure if all the key features are listing we have a test case already added below this test suit

`renders all feature cards`